modules/monitoring-granting-user-permissions-using-cli.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * monitoring/monitoring-your-own-services.adoc
+
+[id="granting-user-permissions-using-cli_{context}"]
+= Granting user permissions using CLI
+
+This procedure shows how to grant to users permissions for monitoring their own services using the CLI.
+
+.Prerequisites
+
+* Have a user created.
+* Log in using the `oc` command.
+
+.Procedure
+
+* Run this command to assign <role> to <user> in <namespace>:
++
+----
+$ oc policy add-role-to-user <role> <user> -n <namespace>
+----
++
+Substitute <role> with `monitoring-rules-view`, `monitoring-rules-edit`, or `monitoring-edit`.
++
+--
+** `monitoring-rules-view` allows reading PrometheusRule custom resources within the namespace.
+** `monitoring-rules-edit` allows creating, modifying, and deleting PrometheusRule custom resources matching the permitted namespace.
+** `monitoring-edit` gives the same permissions as `monitoring-rules-edit`. Additionally, it allows creating new scraping targets for services or Pods. It also allows creating, modifying, and deleting ServiceMonitors and PodMonitors.
+--
++
+[IMPORTANT]
+====
+Whichever role you choose, you must bind it against a specific namespace as a cluster administrator.
+====
++
+As an example, substitute <role> with `monitoring-edit`, <user> with `johnsmith`, and <namespace> with `ns1`. This assigns to user `johnsmith` the permissions for setting up metrics collection and creating alerting rules in the `ns1` namespace.
```

Comment:
I'd shorten to, "This assigns `johnsmith` the permissions..."

Reply:
I'll rather leave it like this, to be more explicit that `johnsmith` is a user.